test.py

```diff
@@ -523,12 +523,12 @@ def test_flatten_unicode_spans(self):
 
     def test_get_serif_fonts(self):
         fonts = self.pdf_writer.get_serif_fonts()
-        for font in ['Noto Serif', 'Noto Sans Cherokee', 'Noto Serif CJK SC', 'Noto Serif Hebrew']:
+        for font in ['Noto Serif', 'Noto Sans Cherokee', 'Noto Serif CJK SC', 'Noto Serif Hebrew', 'NotoSansSymbols2', 'NotoSansMath']:
```

Comment:
NotoSansSymbols2 is available in the Noto-unhinted download. We may have to look at how Noto distribute fonts, because https://github.com/notofonts/noto-fonts/tree/master/unhinted is archived.